### ISSUE TITLE
Final Peer Review - jl2872

### ISSUE THREAD

**Issue by LiJenny** [kew96/LoanMachineLearning#15]
The report explores possible prediction methods on the rate to charge a given loan, using data collected from the borrowers. They are working with a big data size, consisting of over 2,000,000 rows and 151 features.

I think directly dropping rows with significant amount of features missing is the right thing to do, given the fact that you have a lot of data available. The value gained from imputing the missing values might not lead to a significant improvement to the power of your model. I also found the use of the XGBoost model very interesting and learned from this implementation. Finally, the overall prediction power of the model is very strong, with a very low MAE for the final model.

One potential improvement might be to experiment with different loss functions for the each of the models tried in the report (e.g. or the XGBoost). Even within the same model, a different combination of regularizers and loss functions might lead to even better performance. Also, though using the MAE as a performance measure is sufficient to compare the strength of one model over another, it would also be nice to compare this error size with the range of data, to get an idea of the proportion of error size to the typical data size (i.e. a MAE of 10 on a data set of range 1000 is very good, but it would perform poorly on a data set with range 20). Another possible track to explore is to include more visuals for the performance of the model. For example, plotting the predicted value vs the actual value might lead to more insight to where the majority of the error is coming from (e.g. from outliers, from data in a specific range...)
